controllers/api.js

```diff
@@ -247,4 +247,12 @@ router
                 });
             }
         });
+    })
+
+    .use(function(err, req, res, next){
+        if (err) {
+            console.log(err);
+            res.status(500);
+            res[req.rt]({ error: err });
```

Comment:
I'm no expert in express so I more or less took from the code itself, eg. https://github.com/meddulla/oocharts-server/blob/fix/showapierrors/controllers/api.js#L77 and https://github.com/meddulla/oocharts-server/blob/fix/showapierrors/controllers/api.js#L234